pkg/cli/dockerx/docker.go

```diff
@@ -27,11 +29,32 @@ func DockerWithOut(env map[string]string, args ...string) (string, error) {
 }
 
 func IsRunning(name string) (bool, error) {
-	if name == "" {
-		return false, errors.Errorf("instance name not specified")
+	cli, err := client.NewClientWithOpts(
+		client.FromEnv,
+		client.WithAPIVersionNegotiation())
+	if err != nil {
+		return false, err
 	}
-	str, err := DockerOut("ps", "-q", "-f", fmt.Sprintf("name=%s", name))
-	return str != "", err
+
+	containers, err := cli.ContainerList(context.Background(), container.ListOptions{
```

Comment:
Can we use the app context here ?

Reply:
If one was readily available, yes, I am trying to make the smallest possible change right now. 

I am planning on removing all docker CLI shell-out usage using this API, which is when we can make that change.